### PR TITLE
Login docs

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Login doc tweak](https://trello.com/c/r302g8vN/1383-login-doc-tweak)

File: _reference/login-and-authentication.md [hunk 1]
```diff
@@ -20,10 +20,12 @@ Since Console is a proxy for a Rack, Console stores the encrypted original Rack
 
 ### How do I log out (or switch accounts) using the CLI?
 
-When we talk about being "logged in" means that `~/.convox/auth` contains the hostname in `~/.convox.host` along with the corresponding Console or Rack API key (referred to in some places as a "Rack password").
+Being "logged in" means simply that `~/.convox/auth` contains the hostname in `~/.convox.host` along with the corresponding Console or Rack API key (referred to in some places as a "Rack password").
```
Comment (mwarkentin) [inline _reference/login-and-authentication.md]: * `means simply` -> `simply means` - sounds a bit awkward as is?
* `~/.convox.host` should be `~/.convox/host` I think

Comment (soulshake) [inline _reference/login-and-authentication.md]: `means simply` vs `simply means` --> they sound about the same to my ears, but according to Google: 
  * "simply means" -> About 6,680,000 results (0.67 seconds) 
  * "means simply" -> About 484,000 results (0.80 seconds) 

Alright fine, `simply means` it is :+1: 

Regarding `~/.convox.host` vs `~/.convox/host` -- I stared at this for like 10 seconds, and they looked exactly the same, and I thought you had typo'd or something, and my silly brain didn't see the difference until I ctrl-F'd it ^_^ Thanks!

**Comment by mwarkentin** [comment]
;)

**Pull request merged**